crates/common/src/table.rs

```diff
@@ -52,14 +59,29 @@ impl<V> Table<V> {
     }
 
     /// Get a reference to the resource identified by the specified `key`, if it exists.
-    pub fn get(&self, key: u32) -> Option<&V> {
-        self.tuples.get(&key)
+    pub fn get(&self, key: u32) -> Result<&V, UnknownKeyError> {
```

Comment:
Nit: this change (and the one to `get_mut`) seems strange to me. It seems much more fluent to return `Option`.

Reply:
I would prefer to revert it if it's not too big of a deal. I don't feel very strongly about it, but it definitely distracts from the rest of the PR.

Reply:
I've reverted it.